006-vortexa-tech-challenge/src/materials-pie-chart.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from "recharts";
+import { ChartDatum, RampData } from "./types";
+
+import { ErrorBoundary } from "react-error-boundary";
+import React from "react";
+
+const calculateData = (ramps: RampData) => {
+  const data: ChartDatum[] = [];
+
+  try {
```

Comment:
Not entirely sure why we're wrapping try catch here. What happens if we don't include it? 🤔